.markdownlint.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
is this file required?

Reply:
Just to get rid of warnings when fixing readme files - it will "warn" you about the inline HTML and other things